### ISSUE TITLE
Omitting Barlines in mania is impossible

### ISSUE THREAD

**Issue by nl-tatatat** [ppy/osu-stable-issues#20]
My problem is that the "omit first barline" feature only works with taiko, and not also mania even though they have identical barline usage. 

Whenever you place down a red line, a barline will also be at that spot during gameplay. If you place MANY red lines in a row (for example a variable BPM map), then the gameplay is cluttered with barlines and it can be hard to follow the beat. If you map a 1/4th stream and each note has a barline behind it, it can be very distracting and make it harder to both see the notes, and follow the rhythm. In taiko you are able to click a simple button to fix this, but in mania that same button does nothing? https://i.imgur.com/pbkqhDX.png 

This is a pretty simple fix imo. Enable the already existing button to work in mania as well. Its also not a light issue. In taiko, extra barlines are NOT allowed in ranked maps because of how distracting they are... but in mania there is no solution.

**Comment by peppy** [comment]
Fixed in next Cutting Edge release.
